open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -826,23 +1188,19 @@ PROCEDURE QUALIFYMEMBER
    IF LELISTINPUT="" THEN
     LELISTINPUT="none"
    CALL LISTEXPAND
-   IF LELIST(ACCOUNT:TYPE)=TRUE THEN
+   IF LELIST(ACCOUNT:TYPE)=TRUE OR
+      TESTERRORCODE=503 THEN
     DO
      PRINT "{"
      NEWLINE
      CALL PRINTPROGRAMINFO
-     PRINT "  "+Q+"canOpenSubShare"+Q+": false,"
```

Comment:
Hey Tom, When testing my changes to this PowerOn for Deere, Emily noticed that if the account was an invalid account type, she would get a "Cannot Open Subshare" message, but when she had a warning that matched an account level warning in the parameters, she would get an "Oops" Message. I found it was because this line I am writing the comment on is missing above in the warning check. I wanted to add a comment to it before I really PR'd it so you could add that change. BUT when I got to this point in the new code to write this comment. I notice you set the "canOpenSubShare" to true. Shouldn't it, and the new line I am suggesting, be set to false?

Reply:
@jucarson annnnnnddddd I have no clue at this point as to why it was changed from false to true - maybe a search/replace gone wild thing??  With the changes to this version of the code, however, the 'canOpenSubshare' value should be a moot point.   The error code is being set to a 503 value and the error handling procedure called where the displayErrorMessage is being set. It's the displayErrorMessage which is then being intercepted by the UX and the error message displayed to the end user and program run effectively halted instead of the 'canOpenSubshare: false' trigger. That being said, I did correct the issues you noted for error codes 502/503 to make them consistent.  This did, however, also allow me to catch a type-o related to the max error code (corrected '11' to '511'.  Thx!